.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # Run weekly to keep Rust toolchain changes fresh
+    - cron: '0 0 * * 1'
+
+jobs:
+  multiple_toolchains:
+    name: Stable and Beta tasks
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - beta
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install ${{ matrix.rust }} toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+            toolchain: ${{ matrix.rust }}
+            components: clippy, rustfmt
+      - name: Check formatting
+        run: cargo fmt --all --check
+      - name: Check clippy
+        run: cargo clippy -- -D warnings
+      - name: Run test suite
+        run: cargo test
+      - name: Check docs
+        env:
+          RUSTDOCFLAGS: -D warnings
+        run: cargo doc --no-deps --document-private-items
+
+  # Keeping clippy lints happy across a lot of versions is a nightmare, so only
+  # check and test on MSRV without worrying about warnings
+  msrv_only:
```

Comment:
As noted in the other PR, I don't want to give MSRV guarantees at the moment, so I think we don't need this.

Reply:
If you want I can just bump it to `1.67.1` until it potentially gets revisited that way it stays out of the way. If not then I can just remove it entirely